idea/testData/quickfix/when/addRemainingBranchesMissingLeftBracket.fir.kt

```diff
@@ -0,0 +1,10 @@
+// "Add remaining branches" "true"
+// WITH_RUNTIME
+
+sealed class A
+class B : A()
+
+fun test(a: A) {
+  val i = w<caret>hen (a)
+
+/* IGNORE_FIR */
```

Comment:
Why do these 3 new tests have `IGNORE_FIR` in the `.fir.kt` file?

Reply:
Oops!